### PR TITLE
pybitmessage: init at 0.4.4

### PR DESCRIPTION
**Pull request by jgillich**

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/pybitmessage/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, python, pythonPackages, pyqt4, openssl }:
+
+stdenv.mkDerivation rec {
+  name = "PyBitmessage-${version}";
```
Comment (cillianderoiste) [inline pkgs/applications/networking/instant-messengers/pybitmessage/default.nix]: Please use lowercase for the name.

File: pkgs/applications/networking/instant-messengers/pybitmessage/default.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      --replace "exec python2" "exec ${python}/bin/python" \
+      --replace "/opt/openssl-compat-bitcoin/lib/" "${openssl}/lib/"
+    wrapProgram $out/bin/pybitmessage \
+      --prefix PYTHONPATH : "$(toPythonPath $out):$PYTHONPATH"
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://bitmessage.org/;
+    description = ''
+      The official Bitmessage client
```
Comment (cillianderoiste) [inline pkgs/applications/networking/instant-messengers/pybitmessage/default.nix]: Please put this all on one line.

**Comment by jgillich** [comment]
@cillianderoiste Fixed

**Comment by cillianderoiste** [comment]
Perfect, thanks @jgillich!

**Pull request merged**